.github/workflows/snap.yml

```diff
@@ -1,8 +1,8 @@
 on:
   push:
-    branches: [ "snap" ]
+    branches: [ "main" ]
   pull_request:
-    branches: [ "snap" ]
+    branches: [ "main" ]
```

Comment:
why?

Reply:
No need for a separate snap branch, it seems useless as there's nothing to patch on. Keeping it simple. A new update, change the source-tag